packages/nextjs/src/config/webpack.ts

```diff
@@ -5,26 +5,27 @@ import * as SentryWebpackPlugin from '@sentry/webpack-plugin';
 import {
   BuildContext,
   EntryPointObject,
+  EntryPointValue,
   EntryPropertyObject,
   NextConfigObject,
   SentryWebpackPluginOptions,
   WebpackConfigFunction,
   WebpackConfigObject,
   WebpackEntryProperty,
 } from './types';
-import {
-  SENTRY_CLIENT_CONFIG_FILE,
-  SENTRY_SERVER_CONFIG_FILE,
-  SERVER_SDK_INIT_PATH,
-  storeServerConfigFileLocation,
-} from './utils';
+import { SERVER_SDK_INIT_PATH, storeServerConfigFileLocation } from './utils';
 
 export { SentryWebpackPlugin };
 
+logger.enable();
```

Comment:
Will this enable the logger outside of `debug` mode?

Reply:
Heh. Good point. Will fix.